sourcegraph/sourcegraph
# [SG-41363] Migration from `shema.ts` type usage to graphql-operations - `client/vscode`, `client/web`

## Description
Migrate typing using from `schema.ts` to `graphql-oparations`

## Success criteria

1. All imports from `schema.ts` in, the `vscode` package and `client/web/<regression, stores, user, util>` folders,  are replaced with respective imports from `graphql-operations`.

## Ref
[SG Issue](https://github.com/sourcegraph/sourcegraph/issues/41363)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-41363)

## Test Plan
- Verify that all usage of `schema.ts` types has been migrated to their equivalent typing from `graphql-oparations.ts` file in `client/vscode` package.
- Verify that all usage of `schema.ts` types has been migrated to their equivalent typing from `graphql-oparations.ts` file in `client/web` package for the folders: `regression, stores, user, util`

## App preview:

- [Web](https://sg-web-contractors-sg-41363.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-bicamotmtw.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
